### PR TITLE
Add turkish locale

### PR DESCRIPTION
**Pull request by yagizher**
Meow... meow and meow meow

### CODE REVIEW

**Comment by yagizher** [comment]
I honestly dont know what i did in the last commit. You are free to test it or smth.

File: client/main.lua [hunk 1]
```diff
@@ -17,6 +17,8 @@ Citizen.CreateThread(function()
         Citizen.Wait(1000)
 
         TAC.LaodServerConfig()
+            else
+            print("Whoopsie something went really wrong")
```
Comment (ThymonA) [inline client/main.lua]: Undo this change, we are dealing with a while loop and not an if statement.

**Review by ThymonA — CHANGES_REQUESTED** [review]
Undo this change, we are dealing with a while loop and not an if statement.

**Review by yagizher — COMMENTED** [review]
done

**Comment by ThymonA** [comment]
Thanks for adding this!

**Pull request merged**